### DISCUSSION TITLE
Table encoding error

### DISCUSSION THREAD

**Discussion by KsM00000** [getredash/redash#6393] [Q&A] [answered]
Hi. I have redash Version: 8.0.0+b32245 installed on Ubuntu server 18.04 via docker. My version also has additional settings to connect to Oracle. 
I have faced with the problem of recognizing the Slavic language with encoding CL8MSWIN1251. Query returns question marks instead of the text.
I tried to set environment variable NLS_LANG to AMERICAN.CL8MSWIN1251 yet it didn't help. I also tried to use convert in query to transform text to UTF8 encoding. Nothing helped. Please tell me what else can be done.
![image](https://github.com/getredash/redash/assets/79855848/3f8c4508-804d-4d5a-b1ff-4c37aa484be3)

**Comment by justinclift** [comment]
@snickerjp Any ideas with this one? :smile:

**Reply by KsM00000** [reply, marked as answer]
Hi. I resolved the error. I added NLS_LANG=AMERICAN.UTF8 in the env file and it helped. Kinda strange because we have CL8MSWIN1251 encoding in out database but whatever.

**Reply by justinclift** [reply]
No worries.  Glad you worked it out. :smile:
